community/cypher/docs/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/WhereTest.scala

```diff
@@ -32,8 +32,8 @@ class WhereTest extends DocumentingTestBase {
 
   override val properties = Map(
     "Andres" -> Map[String, Any]("age" -> 36l, "belt" -> "white"),
-    "Tobias" -> Map[String, Any]("age" -> 25l),
-    "Peter"  -> Map[String, Any]("age" -> 34l)
+    "Tobias" -> Map[String, Any]("age" -> 25l, "address" -> "Sweden/Malmö"),
```

Comment:
I think we may need to avoid using extended characters for the data, such as Malmö , as this is not able to be rendered properly in the accompanying diagram of the graph.